lib/index.ts

```diff
@@ -201,17 +201,25 @@ export async function createDriver(options: {extraArgs?: string[]} = {}): Promis
   return newDriver;
 }
 
+let _driverCreationPromise: Promise<WebDriver> | undefined = undefined;
+
 // Start up the webdriver and serve files that its browser will see.
 export async function beforeMochaWebdriverTests(this: Mocha.Context) {
   // If this has already been called, there's nothing to do.
-  if (_driver) { return; }
+  if (_driver || _driverCreationPromise) { return; }
```

Comment:
In case `_driver` is unset, but `_driverCreationPromise` is set, it would be important to wait for that promise to resolve, so that the global `driver` is available when `await beforeMochaWebdriverTests` resolves.

A pattern we use in some place for this is like so:
```
return fooPromise || (fooPromise = createFoo());
```
(BTW, it's generalized, with error-handling, in grist-core's `app/common/AsyncCreate`, thought it's not worth bringing that into this repo just for this.)

Reply:
Nicely spotted, thanks for the context!

Reply:
Pushed a fix!